pulpcore/cli/file/publication.py

```diff
@@ -49,8 +50,12 @@ def publication(ctx: click.Context, pulp_ctx: PulpContext, publication_type: str
     click.option(
         "--version", type=int, help=_("a repository version number, leave blank for latest")
     ),
+    click.option(
+        "--manifest",
+        help=_("Filename to use for manifest file containing metadata for all the files."),
+    ),
```

Comment:
Not technically the same, but OK...

Reply:
I believe we should specify pulp_file >= 1.7.0 for this option. 

Also I don’t see this change reflected in the change log.

Reply:
Oh sorry I thought this was the repository command. Ignore my comment about 1.7.0.